ams/server/src/main/java/com/netease/arctic/server/exception/IllegalTaskException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server.exception;
+
+public class IllegalTaskException extends ArcticRuntimeException {
```

Comment:
Perhaps we don't need to declare an exception class for every scenario; a more general exception might suffice for some common situations.

So I recommend to rename the `DuplicateRuntimeException` to `TaskRuntimeException`. All the general errors encountered during the task processing can all be wrapped using it. We may just need to describe the detail in the error message.

HDYT? @rfyu

Reply:
Agreed.I will make changes.